docs/github.md

```diff
@@ -222,10 +222,10 @@ Log all remote bookmarks that you authored or committed to:
 $ jj log -r 'remote_bookmarks() & (mine() | committer(your@email.com))'
 ```
 
-Log all descendants of the current working copy that aren't on any remote:
+Log all ancestors of the current working copy that aren't on any remote:
 
 ```shell
-$ jj log -r '::@ & ~remote_bookmarks()'
+$ jj log -r '::@ & ~::remote_bookmarks()'
```

Comment:
This can also be written using the difference operator instead of the negation operator (same symbol but infix instead of prefix): `::@ ~ ::remote_bookmarks()`. That can also be written as `remote_bookmarks()..@`. Let's switch to that?